CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+
+If you haven't already noticed, Athena is a Discord bot. If you haven't already, go ahead and visit the [Discord Developers Section](https://discordapp.com/developers) in order to create an application for developing bots. Once you've made an application, open it and go to the 'Bot' section. You will find a token, which will allow you to run your bot. Put this token into the `.env` file. As mentioned above, you should have this file after copying `.env.example` and renaming it to `.env`.
+This step is needed to connect the bot authentication to your instance of the bot.
+
+_Friendly reminder to keep your bot token secret to you. The repo is set up such that you should never need to commit your token._
+
+To test Athena, you'll need to invite the bot to your development server. Copy and paste the following link, replacing **XXXX** with the **client_id** of your bot. The client ID can be found on the [Developers Section](https://discordapp.com/developers). `https://discordapp.com/oauth2/authorize?client_id=XXXX&scope=bot`
+
+#### Ready to test your modifications to Athena?
+
+In your root directory, go ahead and run ```npm start```, and you're ready to go. When the bot is ready, it will tell you by displaying a message in the terminal and your bot will be online in your test server.
+
+#### We have a linter set up
+
+We use [eslint](https://eslint.org/) for code checkups. To run the linter, go ahead and run ```npm run lint```.
+
+#### Running tests
+
+Use ```npm test``` to run unit tests. The tests are good for catching when you've accidentally broken a working feature. However, the tests are not comprehensive, and do not always catch everything - please test on your local server as well.
+
+### Questions, Comments, Concerns <a name = "questions"></a>
+
+- **General questions about MP or Athena?** Head on over to the [The Modern Polymath Discord Server](https://discord.gg/U3UQRnJDGt) for any sort of discussion. If you're looking to discuss development or features, we'd recommend the #code-club  channel.
+- **Development questions** or code problems? Search up your question on Stack Overflow or find general forums if your questions are related specifically to code, but no harm in asking other MP members for discussions, or pinging someone from the MP engineering team.
+- **Check out the FAQ** [below!](#faq)
+
+## Pull Request Guidelines <a name = "pullreqguide"></a>
+
+_Please adhere to the following guidelines._ We want to avoid duplicate issues, reports, features, etc.
+
+1. Search the [Issue Tracker](https://github.com/The-Modern-Polymath/athena/issues) for any relevant threads to what you're looking to create a PR for.
+2. Keep it detailed. We want to know your vision, and the more you give us the better discussion we can have!
+3. Please make sure you have read the [license agreement](LICENSE.md). We're using the [MIT License](https://opensource.org/licenses/MIT)- it's only 3 paragraphs!  
+4. Make a new branch to fork your changes from
```

Comment:
"Fork the main repo and make a new branch for your changes" maybe?

Reply:
Done. Changed all 3 things and ready for another review. 😄